USER_MANUAL.md

```diff
@@ -854,6 +863,11 @@ LDPC | Low Density Parity Check Codes - a family of powerful FEC codes
 
 # Release Notes
 
+## V1.8.XX June 2023
```

Comment:
I'm undecided if we need to release a 1.8.10.1 immediately or if this can wait till 1.8.11. Thoughts?

Reply:
> I'm undecided if we need to release a 1.8.10.1 immediately or if this can wait till 1.8.11. Thoughts?

I think the .1 release is needed asap, this has bitten one person but could easily affect others.

I am now testing the new code, but won't get much time in person until Friday evening UTC.